### DISCUSSION TITLE
Special characters cannot be identified in helloworld example

### DISCUSSION THREAD

**Discussion by BruceQin2016** [openlawlibrary/pygls#454] [Q&A]
I did a setup in vscode-playground and copied the code from helloworld.py to Servers folder.
Then i launched the server by press F5 in vscode-playground side and type this in helloworld.txt but nothing happened
![Untitled](https://github.com/openlawlibrary/pygls/assets/18301796/fae5c3a1-dee7-4d2b-ba0d-97b3949a11f5)

It seems that special characters such as "." "/" "," cannot be identified.
How can i do?

**Comment by BruceQin2016** [comment]
Here is my code of hello_world.py
![Untitled1](https://github.com/openlawlibrary/pygls/assets/18301796/a7e09c77-f1fa-4b4f-b575-2440400449d4)

**Reply by alcarney** [reply]
It looks like the hello world example is missing trigger characters if you change the `@server.feature` decorator to something like
```python
@server.feature(
    TEXT_DOCUMENT_COMPLETION,
    CompletionOptions(trigger_characters=["."]),
)
```
does that improve things?

**Reply by BruceQin2016** [reply]
yes, after i add it , it works fine
![image](https://github.com/openlawlibrary/pygls/assets/18301796/ccfb9f8e-cb20-4171-9036-28dc814a2769)
